src/as/math/Math.ts

```diff
@@ -54,10 +53,11 @@ export function generateUUID(): string {
 	}
 
 	NativeMathf.seedRandom(12345678)
-	var d0: i32 = i32(Mathf.random() * 0xffffffff) //| 0
-	var d1: i32 = i32(Mathf.random() * 0xffffffff) //| 0
-	var d2: i32 = i32(Mathf.random() * 0xffffffff) //| 0
-	var d3: i32 = i32(Mathf.random() * 0xffffffff) //| 0
+
+	var d0: u32 = u32(Mathf.random() * 0xffffffff) //| 0
+	var d1: u32 = u32(Mathf.random() * 0xffffffff) //| 0
+	var d2: u32 = u32(Mathf.random() * 0xffffffff) //| 0
+	var d3: u32 = u32(Mathf.random() * 0xffffffff) //| 0
```

Comment:
I think `|0` was a cast to int in plain JS. F.e., running `5.2356 | 0` in your browser console should show `5`.

Reply:
Looks like the `|0` just floors the value. Is that was the `u32()` cast does too?

Reply:
That is correct. Yes.

Reply:
@jtenner That original code was like this.

https://github.com/mrdoob/three.js/blob/dev/src/math/MathUtils.js#L25-L28

Do you recommend any other way to do it?